src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -81,34 +88,43 @@ foam.CLASS({
           'color',
           'year',
           'city',
-          'discipline',
+          'sport',
           'event',
           'country',
           'gender'
         ];
       }
     },
-    'scroller_'
+    'scroller_',
+    'container_',
+    'scrollExtent',
+    'scrollHeight'
   ],
 
   methods: [
     function tableE(parent) {
       var self = this;
       var canvas = this.Canvas.create(null, parent);
+      canvas.attrs({ height: this.scrollHeight$ });
 
       var scroller = this.ScrollCView.create({
+        borderColor: '#e0e0e0',
+        handleColor: '#3f51b5',
         size: 0,
-        //size: 1000,
-        extent: 10
+        extent$: this.scrollExtent$,
+        height$: this.scrollHeight$
       }, canvas);
       canvas.cview = scroller;
       this.scroller_ = scroller;
 
+      this.window.addEventListener('resize', this.onResize);
+
       parent.on('wheel', function(e) {
         scroller.value += e.deltaY > 0 ? 3 : -3;
```

Comment:
Done. It's now converted to a number of rows and rounded up. It's relatively hard, at least with my mousewheel, to get more than the 3 rows at a time, but it is possible. This also makes scrolling by one row work.